### ISSUE TITLE
panic

### ISSUE THREAD

**Issue by lengyijun** [magiclen/chinese-lunisolar-calendar#5]
```
use chinese_lunisolar_calendar::chrono::prelude::*;
use chinese_lunisolar_calendar::LunisolarDate;
use chinese_lunisolar_calendar::LunisolarYear;
use chinese_lunisolar_calendar::SolarYear;
use chinese_lunisolar_calendar::LunarMonth;
use chinese_lunisolar_calendar::LunarDay;

fn main() {
    let solar_date = LunisolarDate::from_lunisolar_year_lunar_month_day(
        LunisolarYear::from_solar_year(SolarYear::from_u16(2004)).unwrap(),
        LunarMonth::Eleventh,
        LunarDay::Twenty,
    )
    .map(|x| x.to_solar_date());
}
```

**Comment by magiclen** [comment]
Thanks for your report. It should be fixed now.

**Comment by lengyijun** [comment]
fixed in 44a06aa
